health/healthcheck.go

```diff
@@ -0,0 +1,50 @@
+package health
```

Comment:
if we wrap our CORS middleware then the entire file here will be redundant

Reply:
That was one of the ideas , to just update the `x/healthx` package and wrap them there, though they wouldn't satisfy the cors package requirement for the handler type, we'd have to add the same structs there as well. 

If you wish for it to be changed in that repo, I'll do so.